addons/hr_expense/models/hr_expense.py

```diff
@@ -755,6 +756,23 @@ def _get_default_expense_sheet_values(self):
             })
         return values
 
+    @api.model
+    def _get_default_sheet_name(self, expenses_to_report):
+        """ Computes the default name for a new expense sheet from the expenses name or dates """
+        if len(expenses_to_report) == 1:
+            sheet_name = expenses_to_report.name
+        else:
+            dates = expenses_to_report.mapped('date')
+            if False in dates:  # If at least one date isn't set, we don't set a default name
+                return False
+            min_date = format_date(self.env, min(dates))
+            max_date = format_date(self.env, max(dates))
+            if min_date == max_date:
+                sheet_name = min_date
+            else:
+                sheet_name = _("%(date_from)s - %(date_to)s", date_from=min_date, date_to=max_date)
+        return sheet_name
+
```

Comment:
We got a misunderstanding :sweat: 
If you put it on the sheet then
@api.model
def _get_default_sheet_name(self, expenses_to_report):

if you put it on the hr.expense then
def _get_default_sheet_name(self):
and change the content to use self

Reply:
got it, updated the PR :)